test/jsonValidationTest.js

```diff
@@ -0,0 +1,124 @@
+const assert = require('chai').assert;
+const Mocha = require('mocha');
+const fs = require('fs');
+
+describe('Validating Cohorts...', () => {
+    const cohortPath = './Cohorts';
+
+    it('Verifying .cohort files', function(done){
+        let failedList = [];
+        browseDirectory(cohortPath, (error, results) => {
+            results.filter(file => file.substr(-7) === '.cohort')
+                .forEach(file => {                    
+                    validateJsonString(file)
+                });
+            
+                done();
+        });
+    });
+
+    it('Verifying cohort settings files', function(done){
+        browseDirectory(cohortPath, (error, results) => {
+            if (error) throw error;
+            results.filter(file => file.substr(-13) === 'settings.json')
+                .forEach(file => {
+                    validateJsonString(file);
+                });
+            
+                done();
+        });
+    });
+
+    it('Verifying cohort category files', function(done){
+        browseDirectory(cohortPath, (error, results) => {
+            if (error) throw error;
+            results.filter(file => file.substr(-22) === 'categoryResources.json')
+                .forEach(file => {
+                    validateJsonString(file);
+                });
+            
+                done();
+        });
+    });
+});
+
+describe('Validating Workbooks...', () => {
+    const cohortPath = './Workbooks';
+
+    it('Verifying .workbook files', function(done){
+        let failedList = [];
+        browseDirectory(cohortPath, (error, results) => {
+            results.filter(file => file.substr(-9) === '.workbook')
+                .forEach(file => {                    
+                    validateJsonString(file)
+                });
+            
+                done();
+        });
+    });
+
+    it('Verifying workbook settings.json files', function(done){
+        browseDirectory(cohortPath, (error, results) => {
+            if (error) throw error;
+            results.filter(file => file.substr(-13) === 'settings.json')
+                .forEach(file => {
+                    console.log(file);
+                    validateJsonString(file);
+                });
+            
+                done();
+        });
+    });
+
+    it('Verifying workbook category json files', function(done){
```

Comment:
for category and settings, do you want further validation?
they have the right fields, etc.  right now this just validates that they are valid json.  so I could have a categoryResources.json that's literally `[]` and that would be valid json.

Reply:
That would be the next step. We have a lot to add.